app/services/intake_progress_calculator.rb

```diff
@@ -0,0 +1,27 @@
+class IntakeProgressCalculator
+
+  starting_index = QuestionNavigation::FLOW.index(Questions::BacktaxesController)
+  question_steps = QuestionNavigation::FLOW[starting_index..-1]
+  doc_overview_index = question_steps.index(Questions::OverviewDocumentsController)
+  POSSIBLE_STEPS = question_steps.insert(doc_overview_index + 1, *DocumentNavigation::FLOW)
+
+  def get_progress(controller, intake)
```

Comment:
@kayline I had wanted to tell you that I only meant to make this an instance method if we ended up breaking up the code into smaller private methods. This still could be done in which case it can be nice to have an `initialize` method for setting instance variables for controller and intake that can be used everywhere. However, as the code is now I think this could just be a class method in which case you could call it with `IntakeProgressCalculator.get_progress(controller, intake)`. Does that make sense? If not we can chat about it.

Reply:
That makes sense, I can update it